src/assessor_tools.yml

```diff
@@ -1,28 +1,398 @@
+    - role: assessment_tool
+      vars:
+        install_dir: /tools/mitm6
+        pip_packages:
+          - mitm6
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/vysecurity/morphHTA/tarball/master
+        install_dir: /tools/morphHTA
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/worawit/MS17-010/tarball/master
+        install_dir: /tools/MS17-010
+        pip_packages:
+          - impacket
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/noahpowers/nlzr/tarball/master
+        install_dir: /tools/nlzr
+        pip_packages:
+          - requests
+          - requests-toolbelt
+          - Sublist3r
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/PowerShellEmpire/PowerTools/\
+        tarball/master"
+        install_dir: /tools/PowerTools
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/NetSPI/PowerUpSQL/tarball/master
+        install_dir: /tools/PowerUpSQL
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/xorrior/RandomPS-Scripts/tarball/master
+        install_dir: /tools/RandomPS-Scripts
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/Arvanaghi/SessionGopher/tarball/master
+        install_dir: /tools/SessionGopher
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/mdsecactivebreach/SharpShooter/\
+        tarball/master"
+        csharp: yes
+        install_dir: /tools/SharpShooter
+        pip_requirements_file: requirements.txt
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/mubix/shellshocker-pocs/tarball/master
+        install_dir: /tools/shellshocker-pocs
+        pip_packages:
+          - scapy
+        python2: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/SimplySecurity/SimplyEmail/\
+        tarball/master"
+        install_dir: /tools/SimplyEmail
+        # The misspelling here is intentional.  See
+        # https://github.com/SimplySecurity/SimplyEmail/blob/master/setup/requirments.txt.
```

Comment:
Helpful comment! 👍

Reply:
https://github.com/SimplySecurity/SimplyEmail/pull/49 if that ever got merged (probably very unlikely), we would need to undo the customization.